docs/index.html

```diff
@@ -1026,8 +1026,177 @@ <h3 id="col-bleed-y">.col-bleed-y <a href="#col-bleed-y">&#35;</a></h3>
 
             <section class="container">
                 <h2 id="visibility-helpers">Visibility helpers <a href="#visibility-helpers">&#35;</a></h2>
-                <p>You can use some generic visibility helpers that can be applied to any element, not just grid elements. resize the viewport to see them in action. The breakpoints can be changed if you're building the css yourself by building the SCSS files.</p>
-
+                <p>You can use some generic visibility helpers that can be applied to any element, not just grid elements. 
+                    <br>The <code>.hidden-*-up</code> classes hide the element when the viewport is at the given breakpoint or wider, 
+                    the <code>.hidden-*-down</code> classes hide the element when the viewport is at the given breakpoint or smaller and <code>.hidden-*</code>
+                    classes hide the element when the viewport is at the given breakpoint.
+                    <br>The breakpoints can be changed if you're building the css yourself by building the SCSS files.</p>
+                <p>for exemple:
```

Comment:
Should be spelt "example"

Reply:
Excuse my French :sweat:

Reply:
oh man it wasn't an insult! I would have just silently fixed it if I was able to work out how to push to your branch :)

Reply:
Mr Jordan, you're lucky beacause you do not hear his french in real ...

Reply:
> oh man it wasn't an insult! I would have just silently fixed it if I was able to work out how to push to your branch :)

I did not take it as an insult, be reassured. It' was just humor :sweat_smile: because "exemple" is "example" in French... Therefore I said "excuse my French" :laughing: